input/pagecontent/changelog.md

```diff
@@ -9,8 +43,13 @@ This is the log of changes made to the eHealth Implementation Guide.
 #### System operations
 #### Instance operations
 ### Code systems
+- CodeSystems using property deprecated, is updated with definition of the property
 ### ValueSets
+- ValueSets must use display values from CodeSystem
 ### ConceptMaps
+- ConceptMaps using Y/N indicators is changed to use ValueSet/v2-0136 and CodeSystem/v2-0136 instead of v2-0532
+- ConceptMaps must use display values from CodeSystem
+- ConceptMap http://ehealth.sundhed.dk/ConceptMap/conceptmap-obs-code-to-ucum fixed wrong mapping
```

Comment:
@sysMATGS Seems this is duplicated for both 3.5.1 and 4.0.0. Which ones do they belong to?